coalib/misc/StringConverter.py

```diff
@@ -16,11 +16,9 @@ class StringConverter:
     def __init__(self,
                  value,
                  strip_whitespaces=True,
```

Comment:
1) i dont think there should be spaces near operator, according to pep, or else gitmate would tell us?
2) it would be worth another commit for that change @mr-karan , probably